Makefile

```diff
@@ -121,7 +122,7 @@ mock-generate: mockgen ## Generate mock
 
 .PHONY: generate-image-docs
 generate-image-docs:
-	./.github/scripts/launch.sh -c ${GIT_BRANCH} -o ${GH_ORG_NAME} -r ${GH_REPO_NAME} -n ${GIT_USERNAME} -e ${GIT_USEREMAIL}
+	./.github/scripts/launch.sh -c "${GIT_BRANCH}" -o "${GH_ORG_NAME}" -r "${GH_REPO_NAME}" -n "${GIT_USERNAME}" -g "${GH_ORG_USER_NAME}" -e "${GIT_USEREMAIL}"
```

Comment:
It is nice to have a new parameter but now we have a lot. Do we really need to introduce a new one ? GH_ORG_USER_NAME has the same value GH_ORG_NAME. In the future, i don't see why it will be different